src/OpenTelemetry.ClrProfiler.Managed/Configuration/ConfigurationKeys.cs

```diff
@@ -22,10 +22,22 @@ public class ConfigurationKeys
         /// <summary>
         /// Configuration key for the exporter to be used. The Tracer uses it to encode and
         /// dispatch traces.
-        /// Default is <c>"Zipkin"</c>.
+        /// Default is <c>"otlp"</c>.
         /// </summary>
         public const string Exporter = "OTEL_EXPORTER";
 
+        /// <summary>
+        /// Configuration key for the otlp protocol to be used.
+        /// Default is <c>"http/protobuf"</c>.
+        /// </summary>
+        public const string ExporterOtlpProtocol = "OTEL_EXPORTER_OTLP_PROTOCOL";
+
+        /// <summary>
+        /// Configuration key for the otlp protocol to be used.
```

Comment:
I think this doc comment is incorrect, the endpoint information should have information about the URL's needed to reach the collector

Reply:
Fixed, thanks!